### PR TITLE
Migrate ovirt to collection - add botmeta

### PR DESCRIPTION
**Pull request by mnecas**
@thedoubl3j, @anshulbehl, @johnlieske

### CODE REVIEW

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -564,6 +564,7 @@ files:
     maintainers: $team_oracle
     support: community
   $modules/cloud/ovirt/: *ovirt
+    migrated_to: ovirt.ovirt_collection
```
Comment (gundalow) [inline .github/BOTMETA.yml:724]: Hi, we need per file (not just directory here), search for other migrated_to entries in this file.

**Review by gundalow — CHANGES_REQUESTED** [review]

File: .github/BOTMETA.yml (hunk 1 above)
Comment (mnecas) [reply .github/BOTMETA.yml:724]: Thank you for the information. Just send the patch if I should update anything please let me know.

**Review by thedoubl3j — APPROVED** [review]
lgtm, shipit

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
Thanks!

**Pull request merged**